memory_map_manager/code_gen.py

```diff
@@ -12,7 +12,7 @@
 
 
 def _parse_filename(f_arg, d_arg, name_contains):
-    if f_arg is None:
+    if f_arg is not None:
```

Comment:
it is actually supposed to find the first .json file if the f_arg is None.

However this is going through a big overhaul too...

Reply:
Just needed these changes to get the script working :-)